canary/apps/angular/.browserslistrc

```diff
@@ -1,17 +0,0 @@
-# This file is used by the build system to adjust CSS and JS output to support the specified browsers below.
```

Comment:
This `.rc` files are already under `canary/apps/angular/angularcli`, and doesn't need to be duplicated here. Same with the other files.